Misc/NEWS.d/next/Library/2023-05-15-17-22-53.gh-issue-104306.YMiegg.rst

```diff
@@ -0,0 +1 @@
+Fix incorrect comment parsing in the :mod:`netrc` module.
```

Comment:
Could you expand on this and include context that would give an arbitrary changelog reader an idea of how the change might impact them?